src/talkaction.h

```diff
@@ -26,6 +26,7 @@
 
 class TalkAction;
 using TalkAction_ptr = std::unique_ptr<TalkAction>;
+using TalkAction_words = std::vector<std::string>;
```

Comment:
Instead of doing a ```using``` here you should rather convert ```std::string words``` into a vector directly so we are not leaving dead stuff